packages/coil-extension/src/popup/components/views/SettingsView.tsx

```diff
@@ -83,17 +122,21 @@ export const SettingsView = () => {
           }}
         >
           <div>
-            {user?.profilePicture && (
+            {user?.profilePicture ? (
               <ProfilePicture src={user.profilePicture} />
+            ) : (
+              <ProfileIcon>
+                <AccountCircleIcon />
+              </ProfileIcon>
             )}
-            {user?.fullName && (
+            {getPrimaryName() && (
               <Typography variant='h6' align='center'>
-                {user.fullName}
+                {getPrimaryName()}
               </Typography>
             )}
-            {(user?.shortName || user?.email) && (
+            {getSecondaryName() && (
               <Typography variant='subtitle1' align='center'>
-                {user.shortName ?? user.email}
+                {getSecondaryName()}
```

Comment:
It will only render as the primary name if there is no full name AND no short name. It will only render as the secondary name if there IS a short name